internal/bminventory/inventory.go

```diff
@@ -212,6 +161,15 @@ func (b *bareMetalInventory) getUserSshKey(params installer.GenerateClusterISOPa
 		"groups": [ "sudo" ]}`, sshKey)
 }
 
+func (b *bareMetalInventory) getIPs() string {
+	ipArr := strings.Split(strings.TrimSpace(os.Getenv("SERVICE_IPS")), " ")
```

Comment:
I think b.ServiceIPs should already have the value populated because of envconfig. Eg. 
ipArr := strings.Split(strings.TrimSpace(b.ServiceIPs), " ")

Reply:
Done